app/validators/metadata_presenter/autocomplete_item_validator.rb

```diff
@@ -0,0 +1,19 @@
+module MetadataPresenter
+  class AutocompleteItemValidator < BaseValidator
+    attr_reader :autocomplete_items
+
+    def initialize(page_answers:, component:, autocomplete_items:)
+      super(page_answers: page_answers, component: component)
+
+      @autocomplete_items = autocomplete_items
+    end
+
+    def invalid_answer?
+      autocomplete_item_list.blank? ? true : autocomplete_item_list.exclude?(JSON.parse(user_answer))
```

Comment:
we want to avoid returning primitive boolean i think, especially as our other functional calls will already perform this for us. essentially you can just `return` with a guard clause for any times that you need which will mean falsey.

```
return if autocomplete_items[component.uuid].blank?

autocomplete_item_list.exclude?(JSON.parse(user_answer))
```

no primitive return needed then